mlrun/api/utils/clients/iguazio.py

```diff
@@ -505,11 +503,34 @@ def _generate_auth_info_from_session_verification_response(
         if x_unix_uid and x_unix_uid.lower() != "unknown":
             user_unix_id = int(x_unix_uid)
 
+        # get user and group ids from response body
+        response_body = response.json()
+        context_auth = (
+            response_body.get("data", {})
+            .get("attributes", {})
+            .get("context", {})
+            .get("authentication", {})
+        )
```

Comment:
Exactly like you implemented in nuclio, we have in mlrun a `get_in` helper method precisely for cases like this